gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/MapContainerViewModel.java

```diff
@@ -431,6 +434,14 @@ public void onCancelButtonClick() {
     return moveFeaturesVisibility;
   }
 
+  public LiveData<Integer> getAddPolygonVisibility() {
```

Comment:
Isn't this button for adding a vertex?

Incorporating the above suggestion, this would become `isAddPolygonButtonVisible`

Reply:
renamed